README.md

```diff
@@ -684,6 +684,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 
 ### • Sports & Health
 
+* [**Box, Box!**](https://github.com/BrightDV/BoxBox) <sup>**[[F-Droid](https://f-droid.org/packages/org.brightdv.boxbox/)]**</sup>
```

Comment:
Please use the [`…/app/…` link format](https://github.com/offa/android-foss/blob/master/CONTRIBUTING.md#f-droid-links).

Reply:
My bad, didn't notice it